cloud/ec2_util.go

```diff
@@ -408,6 +409,18 @@ func makeBlockDeviceMappings(mounts []MountPoint) ([]*ec2aws.BlockDeviceMapping,
 			if mount.Iops != 0 {
 				m.Ebs.Iops = aws.Int64(mount.Iops)
 			}
+
+			if mount.Throughput != 0 {
+				//aws only allows values between 125 and 1000
+				if mount.Throughput > 1000 || mount.Throughput < 125 {
+					return nil, errors.New("throughput must be between 125 and 1,000")
+				}
+				// This parameter is valid only for gp3 volumes.
+				if *m.Ebs.VolumeType != ec2aws.VolumeTypeGp3 {
```

Comment:
It looks to me like we set this to Gp2 at 407 and don't actually override the default until 427, so this won't ever get hit

Reply:
Also, should use utility.ToStringPtr

Reply:
haha yes. Thanks for spotting that! 
Do you mean FromStringPtr?

Reply:
If yes, then done.